### PR TITLE
[FIX] Fix spelling of "collaborative"

### PR DESCRIPTION
**Pull request by stuartpb**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-channel-settings/client/views/channelSettings.html [hunk 1]
```diff
@@ -94,7 +94,7 @@
 			<div class="rc-user-info__row rc-user-info__row--separator">
 				<div class="rc-switch-double">
 					<div class="rc-switch-double__label  {{equal true value 'disabled'}}">
-						{{_ "Colaborative"}}{{equal default value '*'}}
```
Comment (jack17529) [inline packages/rocketchat-channel-settings/client/views/channelSettings.html]: @stuartpb  and this was issue number ?

Comment (stuartpb) [reply packages/rocketchat-channel-settings/client/views/channelSettings.html]: Do I really need a separate issue for a *one-character spelling fix*?

**Comment by geekgonecrazy** [comment]
@RocketChat/ui I can't tell is this one translated?  If so we probably need the i18n files updated?

**Comment by karlprieb** [comment]
@geekgonecrazy yep, apparently we don't have "Collaborative" key on our i18n files. @stuartpb can you add this key?

**Pull request merged**